versions/3.0.md

```diff
@@ -51,7 +51,12 @@ The HTTP Status Codes are used to indicate the status of the executed operation.
 ### Format
 
 The files describing the RESTful API in accordance with the OpenAPI Specification are represented as JSON objects and conform to the JSON standards. YAML, being a superset of JSON, can be used as well to 
-represent a OAS (OpenAPI Specification) file.
+represent a OAS (OpenAPI Specification) file. For YAML, version `1.1` is supported for OAS documents.
```

Comment:
Ah. This seems actually useful – maybe suggest using a preprocessor to resolve it (thereby eliminating the need to deal with it in the spec, but still allowing to use it for users?)